src/scss/layout/page/_page.scss

```diff
@@ -1,7 +1,71 @@
 @use "../../base/mixins";
+@use "../../tokens/screens" as *;
 
+// remains for backwards compatibility.
 .iati-main {
   @include mixins.page-width-container();
   flex: 1;
   padding-block: 1rem;
 }
+
+.iati-page {
```

Comment:
I think just keeping and expanding on `.iati-main` instead of adding the new `.iati-page` would be clearer for users, and removes the need to maintain two classes for backwards compatibility